go_metrics/metrics/tests/test_graphite.py

```diff
@@ -573,6 +575,24 @@ def handler(req):
         [req] = reqs
         self.assertEqual(req.getHeader('Authorization'), None)
 
+    @inlineCallbacks
+    def test_post_request(self):
+        reqs = []
+
+        def handler(req):
+            reqs.append(req)
+            return '{}'
+
+        graphite = yield self.mk_graphite(handler)
+        backend = self.mk_backend(graphite_url=graphite.url)
+        metrics = GraphiteMetrics(backend, 'owner-1')
```

Comment:
Probably we should abstract these set up lines into a helper function that returns `metrics` and `reqs`.